src/Diagnostics.ModelsAndUtils/DataTableUtility.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Diagnostics.ModelsAndUtils;
+using Diagnostics.ModelsAndUtils.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace Diagnostics.ModelsAndUtils
+{
+    public static class DataTableUtility
```

Comment:
There are 2 classes for DataTableUtility? One in data providers and other in ModelsAndUtils

Reply:
No I moved it, is it still showing up in DataProviders?

Reply:
Oh I think you just removed it from csproj but not deleted it. Its in your PR

Reply:
You are correct. I have deleted it.